packages/v4/src/content/design-guidelines/components/overflow-menu/overflow-menu.md

```diff
@@ -2,4 +2,35 @@
 id: Overflow menu
 section: components
 ---
-An **overflow menu** groups a set of actions in a responsive horizontal list. Actions can be set to persist or collapse into a kebab as the viewport shrinks. Overflow menus are useful in toolbars or other places where a group of actions are required.
+
+An **overflow menu** groups a set of actions in a responsive horizontal list to help declutter an interface. Actions can be set to persist or collapse into a kebab as the viewpoint shrinks. Overflow menus are useful in toolbars or other places where a group of actions is required. 
+
+## Usage 
+Use an overflow menu when additional options are available to the user but there is a space constraint. Common usage for overflow menu's happen when switching a UI from a desktop to a mobile device. 
+
+### Examples
+* Use an overflow menu in a table toolbar to group a number of actions and create visual space. Avoid having more than 3 actions fully displayed within a toolbar.
+
+
+  <img src="./img/toolbaroverflow.png" alt="image showing toolbar overflow" width="1765" />
+
+* Use an overflow menu within a table row when additional actions are available that don’t correspond with a column header.
+
+  <img src="./img/tableoverflow.png" alt="image showing table with overflow" width="2315" />
+* Use an overflow menu within a card component to present additional menu options or provide action links.
+ 
+  <img src="./img/card.png" alt="image showing card overflow" width="370" />
```

Comment:
Idk if this could be causing the whole problem but it looks like the actual image file is called "Card.png" with a capital C, but in this img you're calling it with "card.png" lower case C. I'd just adjust that and see if it makes a difference?